src/components/guest-homepage/Card.js

```diff
@@ -1,32 +1,35 @@
 import { Heading, Text, Grid, VStack, Image } from '@chakra-ui/react';
 import Data from './Data';
-import { Link } from 'react-router-dom';
 
-const Card = () => {
+const Card = ({ handleModal, modal }) => {
     return (
-        <Grid
-            templateColumns={[
-                'repeat(1, 1fr)',
-                'repeat(1, 1fr)',
-                'repeat(2, 1fr)',
-                'repeat(3, 1fr)',
-                'repeat(auto-fill,minmax(350px,1fr))',
-            ]}
-            gap={6}
-            py="50px"
-            px={['15px', '75px', '75px', '75px']}
-        >
-            {Data.map((card, id) => {
-                return (
-                    <Link to={`/details/:${card.id}`} key={id}>
+        <>
+            <Grid
+                templateColumns={[
+                    'repeat(1, 1fr)',
+                    'repeat(1, 1fr)',
+                    'repeat(2, 1fr)',
+                    'repeat(3, 1fr)',
+                    'repeat(auto-fill,minmax(350px,1fr))',
+                ]}
+                gap={6}
+                py="50px"
+                px={['15px', '75px', '75px', '75px']}
+                display={`${!modal ? 'Grid' : 'none'}`}
```

Comment:
Don't use display property for hiding cards - the modal should be displayed over the whole page like on the Figma wireframe. Here's another example of what a modal looks like:

![image](https://user-images.githubusercontent.com/5387195/149087929-3113fe58-08e9-4b3e-b2b8-76f4586c2939.png)